src/js/background/backgroundLogic.js

```diff
@@ -75,6 +75,21 @@ const backgroundLogic = {
     return extensionInfo;
   },
 
+  // Remove container data (cookies, localStorage and cache)
+  async deleteContainerDataOnly(userContextId) {
+    await this._closeTabs(userContextId);
+
+    await browser.browsingData.removeCookies({
+      cookieStoreId: this.cookieStoreId(userContextId)
+    });
+
+    await browser.browsingData.removeLocalStorage({
+      cookieStoreId: this.cookieStoreId(userContextId)
+    });
+
```

Comment:
FYI: removeCache does not work with cookieStoreIds (https://github.com/mozilla/multi-account-containers/pull/2498/files#diff-a3f1b60bcaa9aa659bc7e5bd50fa5109ec7313b7bb363cdceeb3c70a67808ba5R96-R98). 

From the docs: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/browsingData/removeCache
![image](https://github.com/user-attachments/assets/95df9336-dd77-4c70-ac83-f559f17023a8)